test/e2e/nix/build_test.go

```diff
@@ -4,13 +4,25 @@ import (
 	"context"
 	"os"
 
-	"github.com/benchkram/bob/bob"
-	"github.com/benchkram/bob/pkg/filepathutil"
 	"github.com/benchkram/errz"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+
+	"github.com/benchkram/bob/bob"
+	"github.com/benchkram/bob/pkg/filepathutil"
+	nix2 "github.com/benchkram/bob/pkg/nix"
 )
 
+var nixCacheContent = `49ce2a3aa83ef14a:/nix/store/fcd0m68c331j7nkdxvnnpb8ggwsaiqac-bash-5.1-p16
```

Comment:
hmm.. i think this will be maintainance hell. TBH.. what is tested here?

Reply:
I used this to test the contents of the .nix_cache file after build. The tests are pinned to a fixed version, so in theory the contents should always be the same on repeated runs.

Alternatively, I could also test that the file exists only

Reply:
Is the result of this  test only related to one specific bob.yaml input or multiple?

Reply:
Only for one `with_task_dependencies.yaml`

Reply:
All right.. then let's keep it.